frame/balances/module.go

```diff
@@ -31,6 +31,7 @@ type Module struct {
 	Index       sc.U8
 	Config      *Config
 	constants   *consts
+	mdConstants *metadataConstants
```

Comment:
pointer types should only be used with purpose (https://google.github.io/styleguide/go/decisions#pass-values)